app/apphandlers/processorAppHandler.go

```diff
@@ -121,48 +127,60 @@ func (processor *ProcessorApp) StartRudderCore(ctx context.Context, options *app
 			startProcessorFunc := func() {
 				g.Go(func() error {
 					clearDB := false
-					StartProcessor(ctx, &clearDB, enableProcessor, &gatewayDB, &routerDB, &batchRouterDB, &procErrorDB, reportingI, multitenantStats)
+					StartProcessor(ctx, &clearDB, enableProcessor, gwDB, rtDB, brtDB, errDB, reportingI, multitenantStats)
 
 					return nil
 				})
 			}
 			startRouterFunc := func() {
 				g.Go(func() error {
-					StartRouter(ctx, enableRouter, tenantRouterDB, &batchRouterDB, &procErrorDB, reportingI, multitenantStats)
```

Comment:
@kuldeep0020 I find `batchRouterDB` more readable than `brtDB`, what do you think?

Reply:
Okay 😓